runtime/tr.source/trj9/optimizer/SPMDPreCheck.cpp

```diff
@@ -78,7 +78,7 @@ bool SPMDPreCheck::isSPMDCandidate(TR::Compilation *comp, TR_RegionStructure *lo
              }
 
           // unsafe opcode - we will skip LAR and SPMD
-          traceMsg(comp, "SPMD PRE-CHECK FAILURE: found disallowed opcode %s at treetop %p in loop %d\n", comp->getDebug()->getName(node->getOpCodeValue()), tt, loop->getNumber());
+          traceMsg(comp, "SPMD PRE-CHECK FAILURE: found disallowed opcode %s at treetop %p in loop %d\n", comp->getDebug()->getName(node->getOpCodeValue()), node, loop->getNumber());
```

Comment:
`node` is not a treetop though. Would it be better if the trace message said "found disallowed opcode %s at node %p"?

Reply:
That's a good point, I'll modify the message for clarity